unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs

```diff
@@ -20,9 +18,6 @@ public class LoadingScreenView : BaseComponentView, ILoadingScreenView
 
         public event Action<ShowHideAnimator> OnFadeInFinish;
 
-        public static LoadingScreenView Create() =>
```

Comment:
I like how you are removing the static accessors. Maybe we should add it to the code convetions to do for all possible following views?